frappe/email/doctype/auto_email_report/auto_email_report.py

```diff
@@ -201,8 +202,13 @@ def send(self):
 		if not self.format == "HTML":
 			attachments = [{"fname": self.get_file_name(), "fcontent": data}]
 
+		sender = None
+		if self.sender and self.sender_email:
+			sender = formataddr((self.sender, self.sender_email))
+
```

Comment:
this won't conform to these settings in email account

<img width="437" alt="Screenshot 2022-09-09 at 12 17 21 PM" src="https://user-images.githubusercontent.com/32034600/189288474-fe74388d-c088-4643-94c8-eb7203bc21fe.png">

is this what you're intending towards?

Reply:
ah seems it was working in that way by default.